frontend/public/components/utils/resource-log.jsx

```diff
@@ -111,7 +116,14 @@ export const LogControls = ({
               </React.Fragment>
             );
           })}
-        <Button variant="link" isInline onClick={onDownload}>
+        <Button component="a" variant="link" isInline href={logURL} target="_blank">
```

Comment:
I think this one should be a real link instead of a button.

Reply:
Fixed